projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -3492,8 +3492,23 @@
         }
     }
 
+    .igx-grid__tbody-pivot-dimension {
+        > .igx-grid-thead__wrapper {
+            border-bottom: initial;
+        }
+
+        .igx-grid-th {
+            height: 100%;
+        }
+    }
+
     %igx-grid__tbody-pivot-mrl-dimension {
+        > .igx-grid-thead__wrapper {
+            border-bottom: initial;
+        }
+
         .igx-grid-th {
+            height: 100%;
             border-bottom: none;
         }
     }
```

Comment:
Not BEM compliant but will accept for now as the grid styles will be completely rewritten in the coming sprints.